plugins/DuskVerb/tools/tuner/staged_tuner.py

```
@@ -0,0 +1,1330 @@
+    print(f"  Active params: {sorted(active_params)}")
+    print(f"  Trials: {n_trials}   Workers: {workers}   Sampler: CMA-ES")
+    print(f"  Locked params: {len(locked_overrides)} fixed values")
+    sigma0 = 0.2
+    if sigma_scale:
+        # CmaEsSampler takes a single global sigma0 (anisotropic per-axis sigma
+        # is unsupported), so widen from the DECAY scale specifically — using
+        # max() over all axes would unintentionally widen every parameter.
+        s = sigma_scale.get('decay', 1.0)
+        if s > 1.01:
+            sigma0 = min(0.5, 0.2 * s)
+            print(f"  Sigma scaled:  {sigma0:.3f} (global sigma0 from decay ×{s:.2f}; "
```

Ruff (0.15.15)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)